File: gcc/testsuite/gfortran.dg/gomp/pr27573.f90
! PR middle-end/27573
! { dg-do compile }
! { dg-require-profiling "-fprofile-generate" }
! { dg-options "-O2 -fopenmp -fprofile-generate" }

program pr27573
  integer i,j
  j = 8
  !$omp parallel
    print *, "foo"
    do i = 1, j - 1
    end do
  !$omp end parallel
end

File: gcc/testsuite/gcc.dg/gomp/pr27573.c
/* PR middle-end/27573 */
/* { dg-do compile } */
/* { dg-require-profiling "-fprofile-generate" } */
/* { dg-options "-O2 -fopenmp -fprofile-generate" } */

extern int puts (const char *);

int
main (void)
{
  int i, j = 8;
#pragma omp parallel
  {
    puts ("foo");
    for (i = 1; i < j - 1; i++)
      ;
  }
  return 0;
}
